### DISCUSSION TITLE
Mermaidで'・'が使えない？

### DISCUSSION THREAD

**Discussion by nnocchi** [increments/qiita-discussions#620] [要望・アイデア・不具合]
<img width="442" alt="image" src="https://github.com/increments/qiita-discussions/assets/45377300/6eec252a-0173-4a38-8e82-458b03789b94">

Notionのmermaidでチャートを作ってQiitaに貼り付けたらエラーが出て表示されなくなりました。
おそらくですが、mermaidのバージョンが古いことに起因するのかな？とも思いました。
どうやら`・`という文字列を入れると表示されなくなるようです。
以下のようにすることで表示できるようになりました^^

Notionのmermaidのバージョンがわからなかったので細かく検証していないので申し訳ないのですが、ここに書いてみました。

```diff
 flowchart TB
	subgraph GitLab
	  gitlab-1[リポジトリ作成] --> gitlab-1-2[環境変数の設定]
	end
    gitlab-1-2 -.- git-1
  	subgraph Dockerコンテナ
	  git-1[git clone ...]
	  git-1 -.- a
		subgraph DBコンテナ
	  a[初期mysqlファイル] --> a2[コンテナ起動]
		a2 --> a3[DB内に初期データが入る]
		end
		a3 -.- |depends on|b
		subgraph Wordpressコンテナ
		  b[Wordpress起動] --> b2[テーマ編集作業]
		  b2 --> b3[Gitでcommitまで]
		end
	end
	b3 -.- |push|gitlab-2
	subgraph GitLab
-	  gitlab-2[CICDで自動デプロイ・スタート]
+	  gitlab-2[CICDで自動デプロイスタート]
	end
	gitlab-2 -.- |CICD|d
	subgraph レンタルサーバー
	  d[ファイルの書き換え]
	end
```

**Comment by tanaka-asami415** [comment]
@inomaru 
ご不便をおかけして申し訳ありません。

ご指摘の通り、バージョンが古いことが原因のようです。
`・` は 10.3.1以降から使えるようですが、Qiitaのバージョンは 10.2.4 となっております。
バージョンアップの対応も予定しておりますが、完了まではお手数をおかけしますが `・` のご使用を避けていただけると幸いです🙇

**Comment by nnocchi** [comment]
@tanaka-asami415 
- ご回答ありがとうございます🙇
- 承知いたしました！
- 「Notionでドラフト -> Qiitaへ投稿」の流れで使用しておりましたので、いつの日か来るアップデート楽しみにしております！
- closeさせていただきますね！

**Reply by tanaka-asami415** [reply]
@inomaru 
お待たせいたしました！
Mermaidのバージョンを10.2.4から10.9.0に変更いたしましたので、`・` を使っていただけるようになりました！
ぜひご活用ください✨
https://qiita.com/release-notes/2024-03-12

**Reply by nnocchi** [reply]
@tanaka-asami415 
ありがとうございます！！  
だいぶ間が空いてしまって申し訳ありません！
素早き対応に感動です・・・笑
